src/error.rs

```diff
@@ -46,6 +95,24 @@ pub enum StorageSetupError {
     NullDestination,
 }
 
+#[cfg(feature = "std")]
+impl std::error::Error for StorageSetupError {}
+
+#[cfg(feature = "std")]
+impl Display for StorageSetupError {
```

Comment:
Ditto re: unnecessarily flagged-out bit of usefulness

Reply:
I'm going to stop marking this pattern in GH and leave its find-and-cleanup as an exercise to the implementer.